### PR TITLE
Upgrade to Cordova 3.9.2

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFOAuthPlugin/SalesforceOAuthPlugin.m [hunk 1]
```diff
@@ -60,18 +60,6 @@ + (NSDictionary *)authErrorDictionaryFromError:(NSError *)error authInfo:(SFOAut
 // ------------------------------------------
 @implementation SalesforceOAuthPlugin
 
-/**
- This is Cordova's default initializer for plugins.
- */
-- (CDVPlugin *)initWithWebView:(UIWebView *)theWebView
```
Comment (bhariharan) [inline libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFOAuthPlugin/SalesforceOAuthPlugin.m]: ```initWithWebView``` is deprecated and gone in ```4.0```. ```pluginInitialize``` is the new way to add plugin specific functionality.

File: libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m [hunk 2]
```diff
@@ -167,7 +167,7 @@ - (void)viewDidLoad
 {
     NSString *hybridViewUserAgentString = [[self class] sfHybridViewUserAgentString];
     [SFSDKWebUtils configureUserAgent:hybridViewUserAgentString];
-    [[self class] overrideCdvOriginalUserAgent:hybridViewUserAgentString];
+    self.baseUserAgent = hybridViewUserAgentString;
```
Comment (bhariharan) [inline libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m]: They have added a cleaner way to supply a user agent.

**Comment by khawkins** [comment]
Looks pretty awesome to me.

**Comment by wmathurin** [comment]
:+1:

**Comment by kchitalia** [comment]
:+1:

**Pull request merged**